### DISCUSSION TITLE
Everything in the TTF Names tab of Font Info is doubled in the generated font file

### DISCUSSION THREAD

**Discussion by jamessouth** [fontforge/fontforge#5181] [General]
If you `cat` out a generated `.ttf` file in the terminal, every field in TTF Names prints twice, like this: `Version 001.000Version 001.000`. Is that OK or is this a bug?

Thank you!

**Comment by ctrlcctrlv** [comment]
Use `ttx` not `cat`, to verify the structure of a font. It's fine and intended.

**Reply by jamessouth** [reply]
ok thank you!
